airbyte-integrations/connectors/source-confluence/metadata.yaml

```diff
@@ -2,24 +2,26 @@ data:
   allowedHosts:
     hosts:
       - ${subdomain}.atlassian.net
+  registries:
+    oss:
+      enabled: true
+    cloud:
+      enabled: true
   connectorSubtype: api
   connectorType: source
   definitionId: cf40a7f8-71f8-45ce-a7fa-fca053e4028c
-  dockerImageTag: 0.1.3
+  dockerImageTag: 0.1.0
   dockerRepository: airbyte/source-confluence
   githubIssueLabel: source-confluence
   icon: confluence.svg
   license: MIT
   name: Confluence
-  registries:
-    cloud:
-      enabled: true
-    oss:
-      enabled: true
+  releaseDate: TODO
```

Comment:
Can you update the release date with 2021-11-05? (This is when the connector was first published based on our docs)